src/converter.js

```diff
@@ -44,12 +44,25 @@ function getReferenceName (referenceName, buildingInputType) {
 }
 
 function mapBasicAttributeType (type, attributeName) {
-  switch (type) {
+  let processedType;
+  if (type instanceof Array) {
+    if (type.length === 2) {
+      switch (type.includes('null')) {
```

Comment:
This switch feels a lot like a `if`. What's the advantage of using a switch here? I'm afraid people will be thrown off by the uncommon style

Reply:
I see your point. In hindsight, I'm not sure why I left that there in the first place.